tasks/src/main/java/com/redhat/parodos/tasks/TibcoWorkFlowTask.java

```diff
@@ -0,0 +1,74 @@
+// @formatter:off
+package com.redhat.parodos.tasks;
+
+import com.redhat.parodos.workflow.task.BaseWorkFlowTask;
+import com.redhat.parodos.workflow.task.WorkFlowTaskOutput;
+import com.redhat.parodos.workflow.task.parameter.WorkFlowTaskParameter;
+import com.redhat.parodos.workflow.task.parameter.WorkFlowTaskParameterType;
+import com.redhat.parodos.workflows.work.DefaultWorkReport;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import com.tibco.tibjms.TibjmsConnectionFactory;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.jms.*;
+import java.util.LinkedList;
+import java.util.List;
+
+@Slf4j
+public class TibcoWorkFlowTask extends BaseWorkFlowTask {
+
+	private Tibjms service;
+	private String url;
+	private String caFile;
+	private String username;
+	private String password;
```

Comment:
I'd suggest to make these `private final`

Reply:
@gciavarrini fixed